set-base-url.py

```diff
@@ -0,0 +1,12 @@
+import fileinput
+
+
+def main():
+    file_path = "data/base.yaml"
+    for line in fileinput.input(file_path, inplace=True):
+        replace_url = line.replace("docs.ansible.com/", "stage-docsite.readthedocs.io")
```

Comment:
Shouldn't this be
```suggestion
        replace_url = line.replace("docs.ansible.com/", "stage-docsite.readthedocs.io/")
```
in case anything comes after the `/`?